pkg/controller/installation/core_controller.go

```diff
@@ -1002,19 +1002,24 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 		}
 	}
 
+	// Fetch any existing default FelixConfiguration object.
+	felixConfiguration := &crdv1.FelixConfiguration{}
+	fcErr := r.client.Get(ctx, types.NamespacedName{Name: "default"}, felixConfiguration)
```

Comment:
nit, but can we just use `err` here? It's confusing to have multiply named error vars when logically they aren't needed (unless for some reason it is needed here?)

Reply:
Just a reminder we'll need to update the OSS RBAC since I don't believe it has felixconfig Get in it right now.